.travis.yml

```diff
@@ -21,7 +19,13 @@ matrix:
         - python: 3.5
           env: TOXENV=docs
         - python: 2.7
-          env: TOXENV=py27-pyramid14
+          env: TOXENV=py27-pyramid19
```

Comment:
the tests against pyramid 1.4 are intentional as it ensures our checks here [1] are valid

[1] https://github.com/Pylons/pyramid_debugtoolbar/blob/cec570385486b6c199b91587ea7c050d845ad7a3/setup.py#L25

Reply:
Is there a reason to continue with `pyramid>=1.4`, instead of `pyramid>=1.8`?